models/advoc/train_evaluate.py

```diff
@@ -360,11 +358,6 @@ def infer(fps, args):
       train_dir=None,
       model_type="regular",
       data_dir=None,
-      data_fastwav=False,
```

Comment:
Should probably add a data_cfg default of `../../datacfg/ljspeech.txt`. I think I need to do this too